backend/src/main/java/ai/verta/modeldb/versioning/BlobDAORdbImpl.java

```diff
@@ -457,32 +456,40 @@ boolean childContains(Set<?> list, Set<?> sublist) {
     }
   }
 
-  List<BlobDiff> getAddedBlobDiff(
+  List<ai.verta.modeldb.versioning.BlobDiff> getAddedBlobDiff(
```

Comment:
that's a good idea. Now when I have tried to change filenames (I thought it will be easier) it appears that I should make some changes in model.java, but also in the generation script. I will spend some time to understand how to do it, but if it will be too hard, I'll postpone it and ask for your help.

Reply:
So I can't just change --file-suffix from java to Model.java in swagger_codegen.sh ? It always expects just java?
I have tried to make corresponding changes in swagger_codegen.sh api.java, model.java, but it always generates the wrong files. Did I forget anything?